### PR TITLE
fix: Return FileError when the underlying tool returns fatal for a file CY-2855

### PR DESCRIPTION
**Pull request by machadoit**
The tool was returning wrongly, it should return a  “FileError”  on those cases. With the proposed fix, will go from:

```json
{
  "file":"alert-gdpr-disclaimer.md",
  "line":1,
  "message":"Command failed: git remote -v\n/bin/sh: git: not found\n",
  "patternId":"null-null"
}
```

to:

```json
{
  "file":"alert-gdpr-disclaimer.md",
  "message":"Command failed: git rev-parse --show-cdup\n/bin/sh: git: not found\n"
}
```

### CODE REVIEW

**Review by franciscodua — COMMENTED** [review]
Can we add a multiple test for this scenario? I'm not sure if we can add tests for errors, but it would be great

**Comment by machadoit** [comment]
Not sure if we ever added that capability, at least before, we couldn't flag "FileErrors" since they ultimately are not issues reported on the file, it is something that worker will know that he needs to ignore for deltas and fixed issues and whatnot, is to mark the file with failure.

I would suggest if that is interesting that we consider adding mechanism on a separated task from this one. What do you think?

File: README.md [hunk 1]
```diff
@@ -36,6 +36,24 @@ You can follow the instructions there to make sure your tool is working as expec
 yarn run docs:gen
 ```
 
+## Limitations
+
+- `remark-validate-links` is only supported with `repository: false`, details
```
Comment (rtfpessoa) [inline README.md:41]: Codacy found an issue: [[write-good] "validate" is wordy or unneeded](https://app.codacy.com/gh/codacy/codacy-remark-lint/pullRequest?prid=6212959)

Comment (rtfpessoa) [inline README.md:41]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/codacy/codacy-remark-lint/pullRequest?prid=6212959)

**Comment by machadoit** [comment]
There were a couple wrong things, but the bottom line was added to the Readme, we have a limitation on a plugin that uses git remote commands to work

**Review by franciscodua — APPROVED** [review]

**Pull request merged**